validators/watchdog.adoc

```diff
@@ -0,0 +1,121 @@
+= Watchdog
+:doctype: article
+:icons: font
+:toc: macro
+
+ifdef::env-github[]
+
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+
+toc::[]
+
+endif::[]
+
+== Introduction
+
+SKALE watchdog is microservice running on every SKALE node for providing SLA agents with node performance metrics. It also can be used for external monitoring goals (validators can use it in their own monitoring tools).
+
+Watchdog is implemented as a Python script running in docker container with Flask webserver that provides simple REST JSON API available on port 3009 (http://YOUR_SKALE_NODE_IP:3009). Currently watchdog can provide data about all SKALE node docker containers, healthchecks of sChains and SGX server, ethereum node endpoint status and hardware information.
```

Comment:
to have the watchdog available to query, Monitoring should be enabled during the node set up in the environment variables.

Reply:
> to have the watchdog available to query, Monitoring should be enabled during the node set up in the environment variables.

it's not correct - watchdog works regardless of any environment variables